src/components/Header/_variables.scss

```diff
@@ -0,0 +1,18 @@
+////
+/// Header variables.
+/// @group header
+/// @copyright IBM Security 2019
+////
+
+@import '@carbon/layout/scss/spacing';
+@import 'carbon-components/scss/globals/scss/layout';
```

Comment:
Hadn't come across a `_variables.scss` partial before. Interesting. Is this something we're using elsewhere?